pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
+
+`procs --sort{{d}}|--sort{{a}} cpu`
```

Comment:
```suggestion
`procs --sort{{a|d}} cpu`
```

Reply:
That syntax was originally my suggestion. Both versions are weird, but when comparing the output I definitively prefer the way it's now:
![image](https://user-images.githubusercontent.com/50295997/127815952-a5e616cc-8156-4fbc-8781-63f79d674b1b.png)

Reply:
The `a` and `d` are part of the argument name, so I think the correct is:
```suggestion
`procs {{--sortd|--sorta}} cpu`
```